src/components/RecoveryPhraseInput.tsx

```diff
@@ -134,10 +136,14 @@ export default function RecoveryPhraseInput({
 const styles = StyleSheet.create({
   container: {
     padding: 0,
-    backgroundColor: 'rgba(103, 99, 86, 0.1)',
+    backgroundColor: colors.white,
   },
```

Comment:
To prevent the content from jumping when the last word is entered.

```suggestion
    borderColor: colors.white,
    borderWidth: 1,
  },
```